packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -70,6 +79,7 @@ function initWebex(e) {
       logger: {
         level: 'debug'
       },
+      services,
```

Comment:
You can simplify this and inline it

```suggestion
          ...(!document.getElementById("production").checked && {
	      services: {
	        discovery: {
	          u2c: "https://u2c-intb.ciscospark.com/u2c/api/v1",
	          hydra: "https://apialpha.ciscospark.com/v1/",
	        },
	      },
	    })
```